src/module/utilityFunctions.js

```diff
@@ -0,0 +1,71 @@
+import Task from './Task.js';
+import * as Elements from './constElements.js';
+
+const displayContent = () => {
+  Elements.taskList.innerHTML = '';
+  Task.TaskObject.forEach((obj) => {
+    Elements.taskList.innerHTML += `
+      <div class="taskDynamic border-bottom  m-0 px-3 py-0 d-flex align-items-center justify-content-between" data-id=${obj.id}>
+              <div class="form-check mb-0 d-flex align-items-center justify-content-start">
+                <input class="form-check-input border" type="checkbox" value="" id="flexCheckDefault checked">                
+                <label class="form-check-label p-3 m-0 d-flex align-items-center justify-content-start" for="flexCheckDefault">
+                  <span class="description h5 m-0 p-0" contenteditable=true>${obj.description}</span>
+                </label>
+              </div>
+              <button class="three-dots h5 btn m-0 icon">
+                <i class="fa-solid fa-ellipsis-vertical"></i>
+              </button>
+              <button class="trash h5 btn m-0 icon hide">
+                <i class="fa-solid fa-trash-can"></i>
+              </button>
+            </div>
+      `;
+  });
+};
+
+const checkLocalStorage = () => {
+  if (JSON.parse(localStorage.getItem('TASKS_LIST')) != null) {
+    Task.TaskObject = JSON.parse(localStorage.getItem('TASKS_LIST'));
+    displayContent();
+  }
+};
+
+const addTask = () => {
+  Task.add(Elements.taskInput);
+  checkLocalStorage();
+};
+
+const reorderTaskObjectId = (obj) => {
+  obj.forEach((item, index) => {
+    item.id = index;
+  });
+};
```

Comment:
- [ ]  it would be better if the index should start from 1 not 0 so if there are 4 tasks the last one should have `index: 4` as stated in the requirements 
<img width="1078" alt="Screen Shot 2022-09-29 at 18 20 27" src="https://user-images.githubusercontent.com/61790587/193099984-43ffc811-a9cd-482c-8b35-2c5f97555bbc.png">